gotv_transportation.md

```diff
@@ -0,0 +1,169 @@
+
+Vehicles are used to transport riders, and have an associated driver. Vehicles and drivers both have various state capabilities and capacities. For example, an SUV may have capacity for 4 riders and be wheelchair accessible. It's driver may be a Spanish speaker and male. In some cases, a vehicle may be operated by more than one person, such as when two people volunteer to share the load of driving, or to feel more comfortable in picking up and traveling with strangers. In these cases, we do not attempt to model the pair of drivers, but do expect the driver to adjust the seating capacity of the vehicle in order to account for the extra driver.
+
+For more details on how a system might be built on the data types proposed here, see the [Scenarios](#scenarios).
+
+## Fields
+
+### Demand Fields
+
+| Name  | Type  | Description
+|-----  |-----  |------
+| availability  | Interval[]  | The Intervals during which this Demand can be satisfied
+| locations | Location[] | The locations to which the associated Riders may be transported
+| riders  | Rider[] | The Riders generating this Demand
+| coordinating_rider | Rider* | If set, then communications about an assigned Ride should be directed towards this Rider. It is an error if this value is set but is not associated with an element of `riders`
+
+Systems may reject creation of demands which would result in a rider being associated with multiple demands.
+
+### Rider Fields
+
+| Name | Type | Description
+|------|--------|------
+| id | string | The unique ID for this Rider
+| person | Person* | If set, then this Rider is associated with a known [Person](people.html).
+| assistant | Person* | If set, then this Rider is not the principal source of the associated Demand, but requires transport along with the one who is in order to help that person through the voting or transportation process. Examples might include healthcare or nursing aides or translators.
+| minor | int* | If set, the Rider is a minor accompanying another Rider in the group, and has the age indicated. As this Rider is a minor, the system has no expectation or desire that they be personally identified as an instance of `Person`.
+| restrictions | RiderRestriction[] | Any Rider restrictions which should be considered when creating an Assignment for this Rider
+
+Note that of the fields `person`, `assistant`, and `minor`, exactly one must be set in each instance.
```

Comment:
For `minor` it says: “exactly one must be set in each instance” but it might be helpful to have the capacity for more than one ‘minor’ to travel with a rider.

Reply:
This is just the single `Rider` here. We would model 3 minors as three separate `Rider` instances, with this field set, in the associated `Demand.Riders` array (see `Demand` above). LMK though if you mean something else.